<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>readFromFile</class>
 <widget class="QDialog" name="readFromFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Gill Sans&quot;;</string>
      </property>
      <property name="text">
       <string>导入处理文件路径：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_readFromFile"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="readFromFile_ok_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readFromFile_cancel_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>欢迎使用中文分词系统！</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>保存成文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_saveFilePath">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="warning_info">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
